Source/Core/Core/ReliableUDPManager/ReliableUnitTest.cpp

```diff
@@ -0,0 +1,785 @@
+// Copyright 2013 Dolphin Emulator Project
+// Licensed under GPLv2
+// Refer to the license.txt file included.
+
+
+#include "ReliableUnitTest.h"
+#include "Common\MsgHandler.h"
+
+ReliableUnitTest::ReliableUnitTest(sf::IpAddress adr, bool server)
+	: m_adr(adr)
+	, m_port(2660)
+	, m_token(0)
+	, m_recvToken(0)
+	, m_controlSock()
+{
+	m_socket = std::make_shared<sf::UdpSocket>();
+	m_socket->bind(2660);
+
+	m_udp = std::make_shared<ReliableUDPConnection>(m_socket, m_adr, 2660);
+	m_socket->setBlocking(true); 
+
+	if (server)
+	{
+		sf::TcpListener listen;
+		listen.listen(2760);
+
+		listen.accept(m_controlSock);
+	}
+	else
+	{
+		m_controlSock.connect(adr, 2760);
+	}
+
+	sendStr[0][0] = "1";
+	sendStr[0][1] = "2";
+	sendStr[0][2] = "3";
+	sendStr[0][3] = "4";
+	sendStr[0][4] = "5";
+
+	sendStr[1][0] = "A";
+	sendStr[1][1] = "B";
+	sendStr[1][2] = "C";
+	sendStr[1][3] = "D";
+	sendStr[1][4] = "E";
+
+
+}
+
+ReliableUnitTest::~ReliableUnitTest()
+{
+	m_socket->unbind();
+}
+
+void ReliableUnitTest::TestsServer()
+{
+	if (!TestConnectionS())
+	{
+		PanicAlertT("Failed");
+		return;
+	}
+	SocketReset();
+
+	/*if (!TestSendingRecievingS())
```

Comment:
Should probably leave a comment indicating why this is commented out, otherwise it should be removed altogether.